common/supplemental/likelySubtags.xml

```diff
@@ -524,14 +524,10 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Michif; ?; ? } => { Michif; Latin; Canada }-->
 		<likelySubtag from="crh" to="crh_Cyrl_UA"/>
 		<!--{ Crimean Tatar; ?; ? } => { Crimean Tatar; Cyrillic; Ukraine }-->
-		<likelySubtag from="crj" to="crj_Cans_CA"/>
```

Comment:
I think the likelySubtags might be normally filtered by the locales currently in CLDR, which would explain these being dropped. There is reason for this, since we don't want thousands of languages in this file.

But @pedberg-icu was the one who added the indigenous language information and might have added these manually, so he should be consulted about this.

Reply:
No, the 4 that got dropped here were all added by John Emmons in 2014 or so. They are dropped because they did not have data in the current version of the territoryInfo for CA; that is fine.

Reply:
Great!